### PR TITLE
Use ocp image streams in knative-build

### PR DESCRIPTION
**Pull request by mgencur**
I'm particularly not sure about `promotion/name`. It was previously knative-v0.3 and we'd like to keep it if possible.

### CODE REVIEW

File: ci-operator/config/openshift/knative-build/openshift-knative-build-release-0.3.yaml [hunk 1]
```diff
@@ -1,12 +1,13 @@
 tag_specification:
-  cluster: https://api.ci.openshift.org
-  name: origin-v4.0
-  namespace: openshift
+  name: '4.0'
+  namespace: ocp
+promotion:
+  name: knative-v0.3
+  namespace: ocp
```
Comment (smarterclayton) [inline ci-operator/config/openshift/knative-build/openshift-knative-build-release-0.3.yaml]: You need to promote somewhere else.  You aren't part of the OCP release.  Change this back to `openshift`.

File: ci-operator/config/openshift/knative-build/openshift-knative-build-release-0.3.yaml [hunk 2]
```diff
@@ -1,12 +1,13 @@
 tag_specification:
-  cluster: https://api.ci.openshift.org
-  name: origin-v4.0
-  namespace: openshift
+  name: '4.0'
+  namespace: ocp
+promotion:
+  name: knative-v0.3
```
Comment (smarterclayton) [inline ci-operator/config/openshift/knative-build/openshift-knative-build-release-0.3.yaml]: You can leave the name whatever you want.

File: ci-operator/config/openshift/knative-build/openshift-knative-build-release-0.3.yaml (hunk 1 above)
Comment (mgencur) [reply ci-operator/config/openshift/knative-build/openshift-knative-build-release-0.3.yaml]: Fixed. Thanks!

**Pull request merged**